crates/typst-cli/src/args.rs

```diff
@@ -386,6 +386,12 @@ pub struct FontsCommand {
 #[derive(Debug, Clone, Parser)]
 pub struct FontArgs {
     /// Adds additional directories to search for fonts
+    ///
+    /// The environment variable `TYPST_FONT_PATHS` is used as a fallback if
+    /// `--font-path` is not specified.
+    ///
+    /// Paths specified via `--font-path` and `TYPST_FONT_PATHS` are separated by
+    /// the system's path separator (`:` on Unix-like systems and `;` on Windows).
```

Comment:
I would reword a bit to something like "If multiple paths are specified, they are separated by ...". I think we can skip the "via `--font-path` and `TYPST_FONT_PATHS`".